wp-content/themes/core/assets/css/src/theme/forms/_mixins.pcss

```diff
@@ -156,31 +156,31 @@
 	}
 
 	label {
-		color: var(--form-control-radio-checkbox-label-color);
```

Comment:
@ryanurban , I wonder if we should leverage the typography mixin for labels here?

Reply:
Ideally yes, for sure, for the actual typographic things

Reply:
will update 👍